.prettierignore

```diff
@@ -2,4 +2,6 @@ build/
 dist/
 public/
 .cache/
-jest/
\ No newline at end of file
+jest/
+tsconfig.json
```

Comment:
Haha, love the fact that you had to _disable_ Prettier to get nice formatting here 😆